doc/spec.md

```diff
@@ -3762,9 +3762,8 @@ x.symmetric_difference([3, 4, 5])         # set([1, 2, 4, 5])
 <a id='set·union'></a>
 ### set·union
 
-`S.union(iterable)` returns a new set into which have been inserted
-all the elements of set S and all the elements of the argument, which
-must be iterable.
+`S.union(iterable...)` returns a new set into which have been inserted
+all the elements of set S and each element of the iterable sequences.
 
 `union` fails if any element of the iterable is not hashable.
```

Comment:
I think this should  read "if any argument is not an iterable sequence, or if any sequence element is not hashable."